onnxruntime/test/providers/provider_test_utils.h

```diff
@@ -487,6 +491,14 @@ class OpTester {
     use_determinism_ = use_determinism;
   }
 
+  void AddPrePackedSharedContainerToSessions() {
```

Comment:
Would it be better to rename this to EnableSharingOfPrePackedWeightsAcrossSessions? The addition of the container is the how.

Reply:
done